### DISCUSSION TITLE
回复评论提示 config.handle must be a function

### DISCUSSION THREAD

**Discussion by Uyoahz26** [walinejs/waline#1287] [Q&A] [answered]
回复评论时会提示 config.handle must be a function，有配置邮箱提醒
![image](https://user-images.githubusercontent.com/42881988/185085155-87376190-87c8-4bb3-84bc-cce648f23924.png)

**Comment by lizheming** [comment]
提供下地址，说一下是什么存储服务

**Comment by Uyoahz26** [comment]
https://waline-test-chi.vercel.app/
存储服务是leancloud国际版

**Comment by lizheming** [comment]
新版本引入的一个 Bug，之后我修复下，可以先用 1.19.1 老版本

**Comment by lizheming** [comment, marked as answer]
最新版 1.19.3 已修复，你重新更新下版本应该就可以了

**Reply by Uyoahz26** [reply]
好的 感谢
